src/index.js

```diff
@@ -0,0 +1,41 @@
+import Router from './Router/Router';
```

Comment:
import should be only from `Router` folder

Reply:
I will consider this